src/components/Home/Home.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { AiOutlineMenuFold, AiOutlineSetting } from 'react-icons/ai';
+import { BiMicrophone, BiSearchAlt2 } from 'react-icons/bi';
+import Books from '../Books/Books';
+import './Home.css';
+import './media.css';
+
+const Home = () => (
+  <div className="container">
+    <div className="header">
+      <div className="left-side">
+        <AiOutlineMenuFold className="menu" />
+        <h3>My books</h3>
+      </div>
+      <div className="right-side">
+        <BiMicrophone className="micro" />
+        <AiOutlineSetting className="setting" />
+      </div>
+    </div>
+    <div className="navlink" />
+    <ul className="nav">
+      <li>Home</li>
+      <li>Books</li>
+      <li>About</li>
+    </ul>
+    <img
+      src="https://images.unsplash.com/photo-1524995997946-a1c2e315a42f?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=870&q=80"
+      alt="bg"
+      className="bg"
+    />
+    <div className="band">
+      <p>BOOK WITH STARS AND PRICE</p>
+      <div>
+        <input type="text" placeholder="Search..." />
+        <div className="div-search">
+          <BiSearchAlt2 className="search" />
+        </div>
+      </div>
```

Comment:
- Hey, great job with the project, I admire your effort so far, You've done well. I just wanted to point out something concerning the search. As it's a requirement to add a search functionality so a user could search by a book name, I don't seem to see it working as intended. when a user searches for a book, it should show up what matches the search e.g, the name of the book, but on the contrary, it doesn't work that way. Please, try to add this implementation to meet this part of the project requirement. :+1: